### ISSUE TITLE
create an assets schema (assets.json) with queries and resolvers for assets as data

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#571]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [x] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
Add support for automatically creating an "assets" graph of everything in the _assets/_ directory, which could really be useful and helpful for things like photo galleries.

## Details
Currently as the _assets/_ folder is automatically known and copied by Greenwood on every build, it might be nice to expose any files as data, just like we do for content.  As documented in #570 , this might help ease the burdened, at least at the workspace level, of plugins needing to have to bring too much to the table.

In other words, by using conventions in an _assets/_ directory, e.g.
```etc
src/
   assets/
     gallery1/
       photo1.png
       photo2.png
     gallery2/
       photo1.png
       photo2.png
       photo3.png
```

You could create whole photo galleries or other multi-media pages and content and access it all with GraphQL.  Or using REST, as we would probably also want to generate an [_assets.json_ in the CLI](https://github.com/ProjectEvergreen/greenwood/blob/master/packages/cli/src/lifecycles/graph.js#L139), just like we do for the graph.

**Comment by thescientist13** [comment]
we might end up depcrecating the GraphQL plugin anyway - #1390
